test/BasicsSpec.purs

```diff
@@ -51,7 +52,26 @@ spec = describe "En- and decoding" $ do
       $ roundtrips (Map.fromFoldable [(Stringy "A" /\ "B"),(Stringy "C" /\ "D")])
     it "roundtrips Map with Int newtype keys"
       $ roundtrips (Map.fromFoldable [(Inty 4 /\ "B"),(Inty 8 /\ "D")])
+    it "roundtrips BigInt" do
+      let
+        inputStr = """{ "number":1, "big": 18014398509481982, "smallBig": 10 }"""
+        expected = """{"smallBig":"10","number":1,"big":"18014398509481982"}"""
```

Comment:
```suggestion
        expected = """{"smallBig":10,"number":1,"big":18014398509481982}"""
```

That's what I would *expect*

Reply:
Unfortunately serialisation only as string

Reply:
It's also strange that the order is reversed.

Reply:
Sad this seems quite impossible without third party libraries. Can you add this to the README? That this is an additional feature to simple-json, that it requires a JS dependency and that you can't use it to produce data for APIs that expect non-string-encoded bigints?

Reply:
> It's also strange that the order is reversed.

I don't care to much, it's the same JSON, I would say.